sycl/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 include(AddSYCLExecutable)
 
 set(SYCL_MAJOR_VERSION 5)
-set(SYCL_MINOR_VERSION 0)
+set(SYCL_MINOR_VERSION 1)
```

Comment:
This is only changed with release notes commits

Reply:
I see.
Do you mean I also have to update the release note?
Or should I revert this change and ask someone who changes the release note to uplift the minor version?

Reply:
No, just revert this change. When a new release is about to be tagged, someone will uplift the version once for all changes between two releases.

Reply:
Done. Thanks.